apps/clinical/src/components/forms/observations/ObservationFormsContainer.tsx

```
@@ -46,6 +46,16 @@ import { usePinnedObservationForms } from '../../../hooks/usePinnedObservationFo
 import styles from './styles/ObservationFormsContainer.module.scss';
 import { executeOnFormSaveEvent } from './utils/formEventExecutor';
+import {
+  detectFormChanges,
+  extractVersionFromFormFieldPath,
+  injectMissingDeleteObs,
+  markUnchangedObservations,
+  mergeObservationStatuses,
+  replaceInterpretationRemovedObs,
+  replaceNoteRemovedObs,
+  restoreComplexValues,
+} from '../../../utils/fhir/observationReconciliation';
```

ESLint

[error] `../../../utils/fhir/observationReconciliation` import should occur before import of `../encounterDetails/EncounterDetails`

(import/order)